software/htmly.yml

```
@@ -0,0 +1,13 @@
+name: "HTMLy"
+website_url: "https://www.htmly.com/"
+source_code_url: "https://github.com/danpros/htmly/"
```

```suggestion
source_code_url: "https://github.com/danpros/htmly"
```

Not sure if this is important, however small fix for consistency with other entries